lib/Graph/Nodes.cpp

```diff
@@ -409,6 +411,40 @@ bool ConvolutionNode::verify() const {
                            Kernels_, Strides_, Pads_, Group_);
 }
 
+bool ConvolutionChannelwiseQuantizedNode::verify() const {
+  bool isValid = expectCompareTrue("Only groupwise quantization is supported.",
+                                   getGroupwise(), true, this);
+
+  if (!isValid) {
+    return false;
+  }
+
+  isValid = verifyConvolution(getInput(), getResult(), getFilter(), getBias(),
```

Comment:
This call just checks that the source and dest element types are the same. Do we expect that the `ConvolutionChannelwiseQuantizedNode` will ever operate with float inputs/output?

Reply:
Is the check right below this what you mean?
`checkType(getInput(), ElemKind::Int8QTy, this);`